### PR TITLE
Wrong filename for some 2.5.20 mima excludes

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Review by raboof — APPROVED** [review]
So it still worked with the wrong filename?

File: akka-stream/src/main/mima-filters/2.5.20.backwards.excludes [hunk 1]
```diff
@@ -10,3 +10,6 @@ ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.io.OutputStreamSou
 ProblemFilters.exclude[MissingClassProblem]("akka.stream.impl.StreamSupervisor$PrintDebugDump$")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.fusing.GraphInterpreter.dumpWaits")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.impl.fusing.GraphInterpreterShell.dumpWaits")
+
+# #26246 Add asJava/asScala to RunnableGraph, not to be extended by users
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.javadsl.RunnableGraph.asScala")
```
Comment (patriknw) [inline akka-stream/src/main/mima-filters/2.5.20.backwards.excludes]: how could it have passed mima checks previously? or is it not needed?

**Review by patriknw — APPROVED** [review]
LGTM, but strange...

File: akka-stream/src/main/mima-filters/2.5.20.backwards.excludes (hunk 1 above)
Comment (johanandren) [reply akka-stream/src/main/mima-filters/2.5.20.backwards.excludes]: I was thinking that it got pulled in anyway, but I didn't dig into how the filename matching works.

**Pull request merged**